### ISSUE TITLE
Can support both remote control and touchscreen?

### ISSUE THREAD

**Issue by mackwu828** [rubensousa/DpadRecyclerView#84]
I have a device that supports both remote control and touchscreen

but onTouchEvent return false and can not be override

```
    final override fun onTouchEvent(e: MotionEvent?): Boolean {
        return false
    }
```

**Comment by rubensousa** [comment]
Hey @mackwu828, can you share the model of your device?

This was intentionally disabled since TVs don't have touchscreen and because selection changes wouldn't be triggered anyway. 

How do you see the selection and its alignment change on touch events?

I could consider for now just allowing the default behavior (not overriding `onTouchEvent`) and let clients trigger the selection change themselves. What do you think?

**Comment by mackwu828** [comment]
the model is R32. the device customized by the manufacturer

there is no selection and alignment change on touch events. just display same as RecyclerView.

when i touch screen to scroll,  DpadRecyclerView auto scroll to top several time, then normal again.  any idea about this?
i want it display same as RecyclerView when touch. leanback VerticalGridView also dont support touch event, it can scroll out of screen.

**Comment by rubensousa** [comment]
@mackwu828 was that happening on the latest alpha?

**Comment by mackwu828** [comment]
yes, i use the latest version 1.0.0-alpha04

**Comment by rubensousa** [comment]
@mackwu828 this PR addresses the issue you mentioned: https://github.com/rubensousa/DpadRecyclerView/pull/105

It will be part of `1.0.0-alpha05`, which should be the last alpha release. Thank you for the feedback. Now it should behave even better than `VerticalGridView`

**Comment by rubensousa** [comment]
Released in 1.0.0-alpha05
